### ISSUE TITLE
Access to token when the Task action is defined in another scope

### ISSUE THREAD

**Issue by TPucke** [dotnet/docs#5085]
I'm using the overload Run(Action, CancellationToken).  The example code shows how to handle the cancellation, but the `token` variable appears to be in scope because the action is defined inline and `token` is declared in the outer scope.  How is the token accessed within the action body when the action is defined in another scope, such as in another class.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 53ffa5ba-8804-d68e-50e8-a28c25506dc3
* Version Independent ID: 6d39fed4-8dc0-1907-8d92-9e30c0907369
* Content: [Task.Run Method (System.Threading.Tasks)](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.run?view=netframework-4.7.1#System_Threading_Tasks_Task_Run_System_Action_System_Threading_CancellationToken_)
* Content Source: [xml/System.Threading.Tasks/Task.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Threading.Tasks/Task.xml)
* Service: **unspecified**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by svick** [comment]
No, there is no provided way to access the `CancellationToken` from inside the lambda. Passing it twice is the common usage, if you can't directly access it from the outer scope, like the example does.

**Comment by TPucke** [comment]
Sad to hear that, but OK thanks for letting me know.  As for the subject documentation, it could be helpful then to clarify this by providing a minimal example (using a CancellationToken) where Task.Run is used to run code that is not an in-line lambda, so that the convenient scope of the current example does not obscure this point.
